recipes/executor/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "executor" %}
+{% set version = "21.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "ea8d4e914186d52e3902763be25bd3ae531cd720f0f07e7a211d76f2d6a326cc"
```

```suggestion
  sha256: ea8d4e914186d52e3902763be25bd3ae531cd720f0f07e7a211d76f2d6a326cc
```
Could you please remove the ``"`` from the ``sha256`` ?